flink-python/docs/reference/pyflink.table/catalog.rst

```diff
@@ -250,3 +250,84 @@ A CatalogDescriptor is a template for creating a catalog instance. It closely re
     :toctree: api/
 
     CatalogDescriptor.of
+
+
+Column
+------
+
+Representation of a column in a :class:`~pyflink.table.ResolvedSchema`.
+
+A table column describes either a :class:`pyflink.table.catalog.PhysicalColumn`,
+:class:`pyflink.table.catalog.ComputedColumn`, or :class:`pyflink.table.catalog.MetadataColumn`.
+
+Every column is fully resolved. The enclosed :class:`~pyflink.table.types.DataType`
+indicates whether the column is a time attribute and thus might differ from the original
```

Comment:
nit: I am not sure how to read this sentence, what do we mean by original here? and what might be expected to be different around time attributes.

Reply:
I am not entirely sure what is meant, the docs for this class were just pulled from the Java equivalent https://nightlies.apache.org/flink/flink-docs-stable/api/java/org/apache/flink/table/catalog/Column.html - perhaps it's not necessary to include this in the Python one, since it doesn't really 'enclose' the data type like the Java one does.